core/src/main/java/tech/tablesaw/api/Table.java

```diff
@@ -82,7 +82,7 @@ private Table(String name) {
      */
     protected Table(String name, Column... columns) {
         this(name);
-        for (Column column : columns) {
+        for (final Column<?> column : columns) {
```

Comment:
there's no need to add `final` throughout this file. I tend to avoid using it except for class members or to enable accessing via a closure in accordance with the google style guide (at least that used to be the rule when I was there)